com.github.skylot.jadx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.skylot.jadx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "command": "jadx-gui",
+    "build-options":{
+        "build-args": [
+            "--share=network"
+        ]
```

Comment:
can't have that. it's not allowed (and the build will fail)

Reply:
well I tried without locally but it failed because it seems to download dependencies at build time

Reply:
Or re-use a JAR file if upstream ships one. People usually don't bother with building Java projects from source and I can't blame them.

Reply:
I switched to use the binary zip provided upstream. Not really happy with it, would rather build from source, but keeping the dependencies manually in sync with each update will be a pain to do, that is not what I have time and motivation for. Hope flatpak will support downloading maven/gradle/npm... dependencies one day.

Reply:
@Bleuzen if that's something you want to work on then https://github.com/flatpak/flatpak-builder-tools is a good place to contribute it.

Reply:
there is already this PR for maven that has been languishing

https://github.com/flatpak/flatpak-builder-tools/pull/253